src/main/java/com/laytonsmith/core/functions/StringHandling.java

```diff
@@ -845,6 +846,63 @@ public Construct exec(Target t, Environment env, Construct... args) throws Cance
 	}
 
 	@api
+	public static class char_is_uppercase extends AbstractFunction {
```

Comment:
Add unit tests for this to make sure it works. Particularly, add a unit test that checks for an empty string, a string with more than 1 letter, a lowercase char and an uppercase char.